components/gitpod-protocol/src/protocol.ts

```diff
@@ -634,6 +634,7 @@ export interface PrebuiltWorkspace {
     buildWorkspaceId: string;
     creationTime: string;
     state: PrebuiltWorkspaceState;
+    statusVersion: number;
```

Comment:
Not entirely happy that the statusVersion needs to be exposed in the protocol and sent to clients. Is there a way (without significantly changing the system) to have this data only available to internal components and continue not having it in clients?

Reply:
> continue not having it in clients?

You could :
 1. introduce a sub-shape of that we use in protocol, and a more expanded shape we expose in `GitpodService`
 2. and hand-remove this field when sending it to clients (we do this with some sensitive field of `WorkspaceInstance` in `GitpodServerImpl`)
 3. but I'm not sure it warrant's the effort, now and for this single field. Let's hope make it better with the new API :wink:

Reply:
Thanks, that's what I was thinking as well. Unless there is a strong objection, I'll leave it as is for now.